synapse/lib/cell.py

```diff
@@ -2454,6 +2455,45 @@ def initSslCtx(self, certpath, keypath):
         sslctx.load_cert_chain(certpath, keypath)
         return sslctx
 
+    def _get_user_for_handler(self, handler: s_httpapi.Handler) -> dict:
+        ret = {}
+        if hasattr(handler, 'web_useriden') and handler.web_useriden:
+            ret['user'] = handler.web_useriden
+        if hasattr(handler, 'web_username') and handler.web_username:
+            ret['username'] = handler.web_username
+        return ret
+
+    def _log_web_request(self, handler: s_httpapi.Handler) -> None:
+        # Derived from https://github.com/tornadoweb/tornado/blob/v6.2.0/tornado/web.py#L2253
+        status = handler.get_status()
+        if status < 400:
+            log_method = t_log.access_log.info
+        elif status < 500:
+            log_method = t_log.access_log.warning
+        else:
+            log_method = t_log.access_log.error
+
+        request_time = 1000.0 * handler.request.request_time()
+        status = handler.get_status()
+        summary = handler._request_summary()
```

Comment:
It seems like this method is a simple string format with public attributes; maybe we could recreate that so its stable rather than using a private method?

Reply:
that's easy to inline into our mesg construction